config/default.json

```diff
@@ -372,6 +372,10 @@
     "images": {
       "useExactUrlsNoProxy": false,
       "baseUrl": "https://demo.vuestorefront.io/img/",
+      "useSpecificImagePaths": true,
```

Comment:
Here it should use false as default value and the documentation should then say we need to change the base image URL in the API to have that working.

Reply:
Yes definitely, this was my fault.